site-docker.yml.sample

```diff
@@ -1,6 +1,9 @@
 ---
 # Defines deployment design and assigns role to server groups
 
+- hosts: all
+  tasks: []
+
 - hosts: mons
   become: True
```

Comment:
If we're gathering facts in the beginning we can probably set `gather_facts` to `false` for the rest of the plays so we don't gather facts twice and speed up the processing.